src/drive/mobile/modules/authorization/duck/selectors.js

```diff
@@ -0,0 +1,27 @@
+import { getServerUrl } from 'drive/mobile/modules/settings/duck'
+
+const getProp = (state, key) => {
+  if (
+    state.mobile &&
+    state.mobile.authorization &&
+    state.mobile.authorization[key] !== undefined
+  ) {
+    return state.mobile.authorization[key]
+  }
+  console.warn(`Authorization prop not found: ${key}`)
```

Comment:
if there are `console.{log,warn}` that will stay in the code, please add `// eslint-disable no-console` to avoid useless warnings